include/SFML/Network/IpAddress.hpp

```diff
@@ -182,15 +182,27 @@ class SFML_NETWORK_API IpAddress
     // Static member data
     ////////////////////////////////////////////////////////////
     static const IpAddress None;      ///< Value representing an empty/invalid address
+    static const IpAddress Any;       ///< Value representing any address (0.0.0.0)
     static const IpAddress LocalHost; ///< The "localhost" address (for connecting a computer to itself locally)
     static const IpAddress Broadcast; ///< The "broadcast" address (for sending UDP messages to everyone on a local network)
 
 private:
 
+    friend bool operator <(const IpAddress& left, const IpAddress& right);
```

Comment:
The friend declaration must have the same signature **and linkage specification** as the operator. This includes `SFML_NETWORK_API`.

That's also why the build currently fails on Visual Studio 2015.